ls1intum/Artemis
# `General`: Update contributor information in about page

### Checklist
#### General


- [x] I tested **all** changes and their related features with **all** on a test server.

### Motivation and Context


The About Page is outdated. Many of the contributing developers are missing.
I open the PR with adding myself to the list, but other developers that contributed and finished their thesis can add themselves within this PR as well.

### Description

The page I am talking about is:
`https://artemistest.ase.in.tum.de/about`

It looks like this:

<img width="1254" alt="Bildschirmfoto 2021-11-25 um 10 18 20" src="https://user-images.githubusercontent.com/41366522/143413892-d147a743-fb44-4121-84c4-acf356e466e4.png">

The part `Individual Contributors` should be updated.

### Steps for Testing

1. Log in to Artemis
2. Navigate to https://artemistest.ase.in.tum.de/about
3. Check if there are any errors on this page and newly added developers are listed properly

### Review Progress





#### Code Review
#### Manual Tests
